flexget/utils/imdb.py

```diff
@@ -292,58 +295,65 @@ def parse(self, imdb_id):
         if photo_elem:
-            self.photo = photo_elem['src']
+            self.photo = photo_elem
         else:
             log.debug('No photo found for %s', self.imdb_id)
 
-        original_name_elem = title_overview.find(attrs={'class': 'originalTitle'})
+        original_name_elem = title_wrapper.find('div', {'class': 'originalTitle'})
         if original_name_elem:
+            self.name = title_wrapper.find('h1', {'class': ''}).contents[0].strip()
             self.original_name = original_name_elem.contents[0].strip().strip('"')
         else:
             log.debug('No original title found for %s', self.imdb_id)
 
-        votes_elem = title_overview.find(itemprop='ratingCount')
+        votes_elem = data.get('aggregateRating', {}).get('ratingCount')
         if votes_elem:
-            self.votes = str_to_int(votes_elem.text)
+            self.votes = str_to_int(votes_elem) if not isinstance(votes_elem, int) else votes_elem
         else:
             log.debug('No votes found for %s', self.imdb_id)
 
-        score_elem = title_overview.find(itemprop='ratingValue')
+        score_elem = data.get('aggregateRating', {}).get('ratingValue')
         if score_elem:
-            self.score = float(score_elem.text)
+            self.score = float(score_elem)
         else:
             log.debug('No score found for %s', self.imdb_id)
 
-        meta_score_elem = title_overview.find(attrs={'class': 'metacriticScore'})
+        meta_score_elem = soup.find(attrs={'class': 'metacriticScore'})
         if meta_score_elem:
             self.meta_score = str_to_int(meta_score_elem.text)
         else:
             log.debug('No Metacritic score found for %s', self.imdb_id)
 
-            # get director(s)
-        for director in title_overview.select('[itemprop="director"] > a'):
-            director_id = extract_id(director['href'])
-            director_name = director.text
-            # tag instead of name
-            if isinstance(director_name, Tag):
-                director_name = None
+        # get director(s)
+        directors = data.get('director', [])
+        if not isinstance(directors, list):
+            directors = [directors]
+
+        for director in directors:
+            if director['@type'] != 'Person':
+                continue
+            director_id = extract_id(director['url'])
+            director_name = director['name']
             self.directors[director_id] = director_name
 
         # get writer(s)
-        for writer in title_overview.select('[itemprop="creator"] > a'):
-            writer_id = extract_id(writer['href'])
-            writer_name = writer.text
-            # tag instead of name
-            if isinstance(writer_name, Tag):
-                writer_name = None
+        writers = data.get('creator', [])
+        if not isinstance(writers, list):
+            writers = [writers]
+
+        for writer in writers:
+            if writer['@type'] != 'Person':
```

Comment:
Why not just = person here.. no need for continue?

Reply:
Indentation

Reply:
it still has Indentation either way :P